test/appium/tests/atomic/chats/test_group_chat.py

```diff
@@ -47,28 +66,26 @@ def test_create_new_group_chat(self):
         for chat in (device_1_chat, device_2_chat):
             if chat.user_name_text.text != chat_name:
                 self.errors.append('Oops! Chat screen does not match the entered chat name %s' % chat_name)
-
         self.verify_no_errors()
 
     @marks.testrail_id(3993)
     @marks.critical
     def test_send_message_in_group_chat(self):
-        message_from_device_1 = 'Hello from device 1'
-        message_from_device_2 = 'Hi there! Sent from device 2'
+        messages = ["Hello from device 1", "Hi there! Sent from device 2"]
```

Comment:
you can map messages to chats in dict() instead of creating this list, chat as key and message as value